File: crossecore-java/src/java_/VariableDeclarationExpression.java
package java_;
public interface VariableDeclarationExpression 
 extends Expression,AbstractVariablesContainer
{
	
	public static com.crossecore.ocl.QuickSet<VariableDeclarationExpression> allInstances_ = new com.crossecore.ocl.QuickSet<VariableDeclarationExpression>(VariableDeclarationExpression.class);
	
		
	public static com.crossecore.ocl.QuickSet<VariableDeclarationExpression> allInstances(){
		
		com.crossecore.ocl.QuickSet<VariableDeclarationExpression> result = new com.crossecore.ocl.QuickSet<VariableDeclarationExpression>(VariableDeclarationExpression.class);
		result.addAll(VariableDeclarationExpression.allInstances_);
		
		
		return result;
	}
	
	//public static com.crossecore.ocl.QuickSet<VariableDeclarationExpression> allInstances = new com.crossecore.ocl.QuickSet<VariableDeclarationExpression>(VariableDeclarationExpression.class);
	
	
	Modifier getModifier();
	void setModifier(Modifier value);
	com.crossecore.ocl.OrderedSet<Annotation> getAnnotations();
	
}

File: crossecore-typescript/java_/VariableDeclarationExpression.ts
import {OrderedSet} from "ecore/OrderedSet";
import {Set} from "ecore/Set";
import {Expression} from "./Expression";
import {AbstractVariablesContainer} from "./AbstractVariablesContainer";
import {Modifier} from "./Modifier";
import {Annotation} from "./Annotation";

export interface VariableDeclarationExpression
extends Expression,
AbstractVariablesContainer

{
	
	modifier:Modifier;
	annotations: OrderedSet<Annotation>;
	

}

